src/BreakEven.API/ViewModels/CreateEmployeeVm.cs

```diff
@@ -0,0 +1,50 @@
+using BreakEven.API.Entities;
+using System.ComponentModel.DataAnnotations;
+
+namespace BreakEven.API.ViewModels;
+
+public class CreateEmployeeVm
+{
+    [Required]
+    public string CPF { get; set; }
+
+    [Required]
+    public string FirstName { get; set; }
+
+    [Required]
+    public string SurName { get; set; }
+
+    [Required]
+    public string Sector { get; set; } // could possibly be swapped for a enum
+
+    [Required]
+    public double GrossSalary { get; set; }
+
+    [Required]
+    public DateTime AdmissionDate { get; set; }
+
+    [Required]
+    public bool HasHealthInsurance { get; set; }
+
+    [Required]
+    public bool HasDentalInsurance { get; set; }
+
+    [Required]
+    public bool HasTransportationAllowance { get; set; }
+
+    public Employee ToDomain()
+    {
+        return new Employee()
+        {
+            CPF = CPF,
```

Comment:
Como o nome das propriedades são identicas, não precisa desse de-para do lado esquerdo para o direito nas atribuições

```suggestion
            CPF,
```

Reply:
Tem certeza? Testando isso local da erro.